### ISSUE TITLE
option to not autopublish screen on syphon

### ISSUE THREAD

**Issue by sloev** [danomatika/loaf#8]
this line: 
https://github.com/danomatika/loaf/blob/dc935ae3f4803baec18aec2f657bf8b7c497e7e9/src/ofApp.cpp#L153

autopublishes the screen on every draw call
i like the idea of having a persistent syphon server running even if i make errors in my lua scripting, but i also like the ability to choose which textures i want to publish.

i suggest to let an commandline option decide if the screen should be autopublished or not.

then i would be able to draw all my content in an fbo with huge size and publish it to builtin syphon server`loaf.getSyphonServer():publishTexture(myFbo:getTexture())`  as well as draw a miniature preview on screen.

in short i wouldnt be bound by the app screen size.

I REEEEEALLY LIKE THIS PROJECT by the way, BIG THANKS!

**Comment by danomatika** [comment]
The line you quote is for a wrapper class for Syphon, not a syphon client instance: https://github.com/danomatika/loaf/blob/master/src/Syphon.h

The current behavior is as you request:
* Syphon is off by default, you must explicitly start it either in the script or via the "--syphon" command line option
* If syphon is left off, you can manually create a server and publish either screen or texture yourself
* You  can also create syphon clients to receive textures. I am using this in a current project but am not able to actually read the texture pixels in Lua so far, let me know if you get it to work. Drawing works fine.

Basic overview is in the readme: https://github.com/danomatika/loaf#syphon-support

I suppose I can add a note that says Syphon is off by default but I believe it says that in the help printout.

**Comment by danomatika** [comment]
Additional features could be:

* add ability to create built-in server without starting, then you can grab the instance and configure it manually, maybe something like "--syphon-init" or change "--syphon" to init the instance and "--syphon-start" for explicit server start
* don't publish screen if there is a script error, probably a good idea in a VJ context

**Comment by sloev** [comment]
>The line you quote is for a wrapper class for Syphon, not a syphon client instance: https://github.com/danomatika/loaf/blob/master/src/Syphon.h

i know, i need a syphon server and i need one that is not dropped on each script hotload.

what i meant was that i have a need for exposing a texture that is larger than my window (using an fbo)
and i wanna use the builtin syphon server but i cant since its only able to publish the screen (which it does per automatic as long as the server is not nil)

again THIS SOFTWARE IS COMPLETELY AWESOME!!!!!! BIG THANKS!!!!

ive already done some glitchy stuff with it :-)

https://www.youtube.com/watch?v=muiV5cnEA7o

https://www.youtube.com/watch?v=_rZtMqqKhrU
